content/docs/reference/config/installation-options-version-comparison/index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Installation Options - Generate differences between current release and previous release (v1.0)
+description: Describes the differences between the current and previous releases of installation options from the Helm charts.
+weight: 30
+keywords: [kubernetes,helm]
+---
+
+## `ingress` options
+
+| Old Key | New Key | Old Default Value | New Default Value | Old Description | New Description|
+| --- | --- | --- | --- | --- | --- |
+| `ingress.service.ports.nodePort` | `n/a` | `32000` | `n/a` | `` | `n/a` |
```

Comment:
This is significantly more readable.  I wonder as a future followup if it would be possible to classify keys that were added, vs keys that were removed.  A deletion (`n/a/`) in New Key could be one table, while an addition (`n/a` in old key) could be ea different table.

Reply:
@sdake I have tried to address this in the latest PR. Not sure if this is what you had in mind. Let me know if we need to make any changes. Thanks.

Reply:
@johnma14 I think you are headed in the right direction.  I have added some comments and asked for some information architecture expertise from @rcaballeromx.  Note @rcaballeromx this document is autogenerated from yaml files, so coming up with a good structure to begin with keeps the churn on the generation implementation to a minimum.